### PR TITLE
feat: DEV-1852: Add Inside Label for VideoRectangle

### PR DESCRIPTION
**Pull request by superdev808**

### CODE REVIEW

File: src/tags/object/Video/Rectangle.js [hunk 1]
```diff
@@ -22,45 +31,175 @@ const RectanglePure = ({ reg, frame, workingArea, ...rest }) => {
...
+          <Text
+            text={score.toFixed(2)}
+            fontFamily="-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, 'Noto Sans', sans-serif"
+            fontSize={fontSize}
+            fill="white"
+            padding={2}
+          />
+        </Label>
+      )}
+      {showLabels && (
```
Comment (hlomzik) [inline src/tags/object/Video/Rectangle.js:139]: Let's do this better - move this code to separate component and just use it here

File: src/tags/object/Video/Rectangle.js [hunk 2]
```diff
@@ -22,45 +31,175 @@ const RectanglePure = ({ reg, frame, workingArea, ...rest }) => {
   if (!box) return null;
 
   const { realWidth: waWidth, realHeight: waHeight } = workingArea;
+  const text = reg.getLabelText(",");
+  const showLabels = reg.store.settings.showLabels;
+  const { score } = reg;
+  const fontSize = 20;
+  const height = 30;
+  const ss = showLabels && score;
```
Comment (hlomzik) [inline src/tags/object/Video/Rectangle.js:39]: `showScore`

File: src/tags/object/Video/Rectangle.js [hunk 3]
```diff
@@ -22,45 +31,175 @@ const RectanglePure = ({ reg, frame, workingArea, ...rest }) => {
...
-    width: box.width * waWidth / 100,
-    height: box.height * waHeight / 100,
+    x: (box.x * waWidth) / 100,
+    y: (box.y * waHeight) / 100,
+    width: (box.width * waWidth) / 100,
+    height: (box.height * waHeight) / 100,
     rotation: box.rotation,
   };
 
+  const onClickLabel = reg.onCLickRegion;
```
Comment (hlomzik) [inline src/tags/object/Video/Rectangle.js:115]: typo, L

**Review by hlomzik — CHANGES_REQUESTED** [review]

File: src/tags/object/Video/Rectangle.js (hunk 2 above)
Comment (superdev808) [reply src/tags/object/Video/Rectangle.js:39]: showScore is not existing.

**Pull request closed without merging**